app/views/alchemy/admin/attachments/show.html.erb

```diff
@@ -1,3 +1,66 @@
-<iframe src="<%= alchemy.show_attachment_path(@attachment) %>" frameborder=0 class="full-iframe">
-  Your browser does not support frames.
-</iframe>
+<div class="resource_info">
+  <div class="value">
+    <label>
+      <%= render_icon @attachment.icon_css_class %>
+    </label>
+    <p><%= @attachment.name %></p>
```

Comment:
Here, I prefer to have `@attachment.file_name` instead of the name (that we put in the modal title).